types/defines/ai.d.ts

```diff
@@ -148,15 +169,33 @@ export declare abstract class BaseAiTextGeneration {
   inputs: AiTextGenerationInput;
   postProcessedOutputs: AiTextGenerationOutput;
 }
+export type AiTextToSpeechInput = {
+  prompt: string;
+  lang?: string;
+};
+export type AiTextToSpeechOutput =
+  | Uint8Array
+  | {
+      audio: string;
+    };
+export declare abstract class BaseAiTextToSpeech {
+  inputs: AiTextToSpeechInput;
+  postProcessedOutputs: AiTextToSpeechOutput;
+}
 export type AiTextToImageInput = {
   prompt: string;
+  negative_prompt?: string;
+  height?: number;
+  width?: number;
   image?: number[];
+  image_b64?: string;
   mask?: number[];
   num_steps?: number;
   strength?: number;
   guidance?: number;
+  seed?: number;
 };
-export type AiTextToImageOutput = ReadableStream<Uint8Array>;
```

Comment:
It looks like this was intentionally changed in https://github.com/cloudflare/workerd/pull/2757 because that's what the API returned. Is this no longer the case?